CMakeLists.txt

```diff
@@ -103,7 +102,7 @@ find_package(catkin REQUIRED COMPONENTS
 #add_service_files(DIRECTORY srv FILES AddTwoInts.srv)
```

Comment:
With this PR, can we already remove the unnecessary future use commented lines like 102 or 104/105? What do you think? Im sure it can be easily added back if needed in the future..

Reply:
done in 684ee5ab34a2272bdf4648b499aebf2556701d7c